src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!currentTab) {
+    [currentTab] = tabs;
+  }
```

Comment:
It takes a time to understand what happened here, it works, but it's better to avoid such kind of redeclaration